test/aircraft/FlightManagementSystem/RouteModel.spec.js

```diff
@@ -1032,6 +1062,16 @@ ava('._createAmendedDivergentLeg() calls ._createLegsFromSidWaypointsBeforeWaypo
     t.deepEqual(result, expectedResult);
 });
 
+ava('._createAmendedDivergentLeg() returns the leg unmodified when divergent waypoint is the STAR leg\'s last fix', (t) => {
+    const model = new RouteModel('BCE.GRNPA1.KLAS07R');
+    const waypointName = 'LEMNZ';
+    const legIndex = 0;
+    const expectedResult = [model._legCollection[legIndex]];
```

Comment:
You should specify the expected result inline here instead of relying on the model to produce it for you.

Reply:
done